src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    public static final int DEFAULT_INITIAL_CAPACITY = 16;
+    public static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    public static final int CAPACITY_INCREASER = 2;
+
+    Node<K, V>[] bucketArray;
```

Comment:
let's add access modifiers. And do you really need to make your constants public?